AppCenter/AppCenter.xcodeproj/xcshareddata/xcschemes/AppCenter iOS Distribution.xcscheme

```diff
@@ -15,8 +15,8 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "6E04017D1D1CAC710051BCFA"
-               BuildableName = "AppCenterIOSFramework"
-               BlueprintName = "AppCenterIOSFramework"
+               BuildableName = "AppCenter iOS Distribution"
```

Comment:
`AppCenter iOS Fat Framework` doesn't work? Because that's what this scheme is producing right?